indy_node/test/auth_rule/test_get_auth_rule.py

```diff
@@ -198,32 +199,32 @@ def test_auth_rule_after_get_auth_rule(looper,
     constraint = generate_constraint_list(auth_constraints=[generate_constraint_entity(role=TRUSTEE),
                                                             generate_constraint_entity(role=STEWARD)])
     # get all auth rules
-    auth_rules = sdk_get_auth_rule_request(looper,
-                                           sdk_wallet_trustee,
-                                           sdk_pool_handle)
+    auth_rules_resp = sdk_send_and_check_get_auth_rule_request(
+        looper, sdk_pool_handle, sdk_wallet_trustee
+    )
+    auth_rules = auth_rules_resp[0][1][RESULT][DATA]
+    rule = auth_rules[0]
 
     # prepare action key
-    dict_key = dict(auth_rules[0][1][RESULT][DATA][0])
+    dict_key = dict(rule)
     dict_key.pop(CONSTRAINT)
 
-    # prepare "operation" to send AUTH_RULE txn
-    op = dict(auth_rules[0][1][RESULT][DATA][0])
-    op[TXN_TYPE] = AUTH_RULE
-    op[CONSTRAINT] = constraint
+    # prepare AUTH_RULE request
+    rule[CONSTRAINT] = constraint
+    req_json = build_auth_rule_request_json(
```

Comment:
Here we re-build the request as well, but it would be better to send the result json 'as-is' (just change the constraint)